Migrating-AU-to-Chocolatey-AU.md

```diff
@@ -0,0 +1,21 @@
+# Migrating From `AU` To `Chocolatey-AU`
+
+With the [`1.0.0` release](https://github.com/chocolatey-community/chocolatey-au/releases/tag/1.0.0) of `Chocolatey-AU`, there is a need to migrate to the new package name.
+
+How you migrate from `AU` to `Chocolatey-AU` will depend on how you used `AU` in your repository. Below are some scenarios, if you encounter a scenario not yet covered, please reach out in the [Chocolatey Community](https://ch0.co/community).
+
+## Your Repository Currently Clones The Original `AU` Repository
+
+If you are cloning the [original `AU` repository](https://github.com/majkinetor/au/) and building the module from there, you will want to update to clone the Chocolatey Community `Chocolatey-AU` repository: `https://github.com/chocolatey-community/chocolatey-au.git` instead. Once that has been done, you will want to [update your update scripts](#updating-your-update-scripts) to use the `Chocolatey-AU` PowerShell Module instead of `AU`.
+
+## Your Repository Installs AU From The Chocolatey Community Repository
+
+If you are installing `AU` from the [Chocolatey Community Repository](https://community.chocolatey.org/), you will want to update it to `choco install chocolatey-au --confirm` instead of `choco install au --confirm`. Once that has been done, you will want to [update your update scripts](#updating-your-update-scripts) to use the `Chocolatey-AU` PowerShell Module instead of `AU`.

```

Comment:
```suggestion
If you are installing `AU` from the [Chocolatey Community Repository](https://community.chocolatey.org/), you will want to update it to `choco install chocolatey-au --confirm` instead of `choco install au -y`. Once that has been done, you will want to [update your update scripts](#updating-your-update-scripts) to use the `Chocolatey-AU` PowerShell Module instead of `AU`.

```

Reply:
When writing code, refer to the [Writing Documentation](https://design.chocolatey.org/content-and-marketing/writing-documentation#code-style) guide and the [Scripting and Best Practices Style Guide](https://docs.chocolatey.org/en-us/choco/commands/#scripting-integration-best-practices-style-guide). 

Specifically here, we don't use `--confirm`, we use `-y`.